CRFModuleValidation/Assets.xcassets/closeActivity.imageset/Contents.json

```diff
@@ -10,7 +10,6 @@
     "author" : "xcode"
   },
   "properties" : {
-    "template-rendering-intent" : "template",
-    "preserves-vector-representation" : true
```

Comment:
we don't want this for this one?

Reply:
I realized that I was turning it on for everything, including assets that don't resize, except when they are added to the bundle. Close button, back button, info button are all 32x32 and are independent of device.